ocs_ci/helpers/helpers.py

```diff
@@ -4093,3 +4095,9 @@ def verify_quota_resource_exist(quota_name):
     return quota_name in [
         quota_resource.get("metadata").get("name") for quota_resource in quota_resources
     ]
+
+
+def check_cluster_is_compact():
+    existing_num_nodes = len(node.get_all_nodes())
+    if existing_num_nodes == 3:
```

Comment:
I would suggest adding one more check of node type being both master and worker, in order to make the check of compact mode or not more explicit